packages/deployment/app/cloudformation.template.yml

```diff
@@ -790,7 +791,13 @@ Resources:
         MinInstancesInService: 0
     Properties:
       AvailabilityZones:
+      {{# if ecs.availabilityZones }}
```

Comment:
I like the fact that this change preserve backward compatibility 👏 💯

Reply:
Is there anything we should update the docs?

Reply:
good idea, opened https://github.com/cumulus-nasa/cumulus-nasa.github.io/pull/55/